api/tmdb/build.gradle.kts

```diff
@@ -17,7 +17,7 @@
 
 plugins {
     id("kotlin")
-    alias(libs.plugins.kotlin.kapt)
+    alias(libs.plugins.ksp)
```

Comment:
Did you notice much build time improvement after making this change?

Reply:
Maybe? I don't have any numbers to back that up though. There's no more Java being compiled at all, which definitely helps. The goal was to support KMP though.